### ISSUE TITLE
Screen low-resolution resolution mode

### ISSUE THREAD

**Issue by takahirox** [mozilla/hubs#5282]
**Is your feature request related to a problem? Please describe.**

Rendering is one of the most costly functions. Rendering is slow especially on lower-end devices (having a cheap graphics card). Such device users may think Hubs is useless due to the poor performance.

**Describe the solution you'd like**

Consider lower resolution screen for such devices. Rendering to lower resolution screen should be faster, of course it sacrifies visual quality tho. But some worse performance may be acceptable than very poor performance.

Some ideas.

1. Add a new low-resolution mode in the preferences
3. Use lower-resolution if the quality mode is low
4. Monitor the FPS counter and dynamically adjust the screen resolution
5. Detect the user device and use lower resolution if the device is a mobile or low-end device

**Describe alternatives you've considered**

We have maximum resolution in the preference. Better discoverability with it and advice to set an appropriate values might be another option.

**Additional context**

* Making performance test would help us determine whether this optimization is necessary.
* My impression is that currently the performance bottlenecks are mainly on CPU end, not GPU end. So it would be better to think of this type of optimization that sacrifices the graphics quality after we do CPU optmization well.

**Comment by takahirox** [comment]
Related or duplicated with #2222

**Comment by takahirox** [comment]
Closing in favor of #5466 (I may want to think of auto resolution adjustment for lower end devices after #5466)
